src/include/sof/math/iir_df1.h

```diff
@@ -13,6 +13,7 @@
 #include <sof/common.h>
```

Comment:
Minor: @singalsu "The omitting of outer loop for parallel biquads and check for null coefficients and use of fixed loop count of two" very complex language is, comments Yoda.